ipemu/csrc/vbridge_impl.cc

```diff
@@ -237,6 +241,26 @@ VBridgeImpl::VBridgeImpl()
   auto &csrmap = proc.get_state()->csrmap;
   csrmap[CSR_MSIMEND] = std::make_shared<basic_csr_t>(&proc, CSR_MSIMEND, 0);
   proc.enable_log_commits();
+
+  char *primary_tck_str = get_env_arg("COSIM_tck");
+  tck = std::stod(primary_tck_str);
```

Comment:
what's tck used for?

Reply:
Calculate the ratio between dram clock period and IP clock period.